model/src/pyrenew/observation/negativebinomial.py

```diff
@@ -93,8 +93,9 @@ def sample(
             name=self.name,
             fn=dist.NegativeBinomial2(
                 mean=mu + self.eps,
-                concentration=concentration,
+                concentration=concentration.value,
             ),
             obs=obs,
         )
-        return (negative_binomial_sample,)
+
+        return (SampledValue(negative_binomial_sample),)
```

Comment:
I think we should probably pass `self.t_start` and `self.t_unit` here, even if they're typically going to be `None`.

Reply:
Noted